### DISCUSSION TITLE
The Secret Agent Text

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#17] [Galaxy Trivia] [answered]
Your text wants to go undercover, hidden from sight but still readable by screen readers. How can it blend into the shadows using CSS?

**Comment by mitulpatel224** [comment, marked as answer]
We can set `text-indent` to a negative value to move away the text from the visible part of the screen.

**Reply by meviru** [reply]
Congratulations!! 👏👏 You've just earned the [Galaxy Brain](https://github.com/mitulpatel224?tab=achievements&achievement=galaxy-brain) badge. Keep it up, and remember to give the repository a star to show your support! 🎉
